src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String fileContent = readFileContent(fromFileName);
+        String report = createReport(fileContent);
+        writeToFile(toFileName, report);
+    }
+
+    public static String readFileContent(String filePath) {
```

Comment:
```suggestion
    private String readFileContent(String filePath) {
```

please make helper methods private
and there is no need to make them static